SpatialGDK/Source/SpatialGDKServices/SpatialGDKServices.Build.cs

```diff
@@ -29,8 +29,9 @@ public SpatialGDKServices(ReadOnlyTargetRules Target) : base(Target)
 				"Json",
 				"JsonUtilities",
 				"UnrealEd",
-				"Sockets"
-			}
+				"Sockets",
+                "HTTP"
```

Comment:
This is using spaces rather than tabs

Reply:
This is the most stupid thing I've seen with VS in a long time. I have all languages set to keep tabs. I even disabled some setting called "Adaptive formatting" which apparently "figures out what your file should have", and still when editing c# files every time I write it changes / adds them as spaces. Even if I reformat the whole file to only have tabs, when I add something it becomes spaces.
So for now adding Tabify to at least know when I save what's the deal, but it's the stupidest thing that something so basic isn't working.